src/query/src/dist_plan/merge_scan.rs

```diff
@@ -192,7 +195,12 @@ impl MergeScanExec {
                     header: Some(RegionRequestHeader {
                         tracing_context: tracing_context.to_w3c(),
                         dbname: dbname.clone(),
-                        timezone: tz.clone(),
+                        query_context: Some(QueryContext {
+                            current_catalog: current_catalog.clone(),
+                            current_schema: current_schema.clone(),
+                            timezone: timezone.clone(),
+                            extensions: extensions.clone(),
```

Comment:
```suggestion
                            current_catalog,
                            current_schema,
                            timezone,
                            extensions,
```

Reply:
It seems the test is failed due to this?

Reply:
Yes, revert it. It's in a loop, my mistake.